spec/Cabin/Cabin.vspec

```diff
@@ -121,6 +121,12 @@ DoorCount:
   description: Number of doors in vehicle.
 
 
+IsWindowChildLockEngaged:
```

Comment:
@SebastianSchildt - it was not sufficient to add `instantiate: False` as the signal was in the Window branch which is not instantiated as each door has only one instance. Now moving it up to Cabin level. Please check